apps/web/modules/survey/list/components/survey-card.tsx

```
@@ -50,7 +50,9 @@ export const SurveyCard = ({
 
   const isSurveyCreationDeletionDisabled = isReadOnly;
 
-  const { refreshSingleUseId } = useSingleUseId(survey);
+  const { refreshSingleUseId } = isReadOnly
+    ? { refreshSingleUseId: () => Promise.resolve(undefined) }
+    : useSingleUseId(survey);
```

ESLint

[error] React Hook "useSingleUseId" is called conditionally. React Hooks must be called in the exact same order in every component render.

(react-hooks/rules-of-hooks)